substrate/utils/frame/benchmarking-cli/src/pallet/mod.rs

```diff
@@ -166,6 +167,21 @@ pub struct PalletCmd {
 	)]
 	pub wasmtime_instantiation_strategy: WasmtimeInstantiationStrategy,
 
+	/// Optional runtime blob to use instead of the one from the genesis config.
+	#[arg(long, conflicts_with = "chain")]
+	pub runtime: Option<PathBuf>,
+
+	/// Do not fail if there are unknown but also unused host functions in the runtime.
+	#[arg(long)]
+	pub allow_missing_host_functions: bool,
+
+	/// How to construct the genesis state.
+	///
+	/// Uses `GenesisBuilder::Spec` by default and  `GenesisConstructor::Runtime` if `runtime` is
+	/// set.
+	#[arg(long, value_enum)]
+	pub genesis_builder: Option<GenesisBuilder>,
```

Comment:
I feel like this can already be derived from `runtime`, no? 

`--runtime` implies `GenesisBuilder::Runtime`
`--chain` implies `GenesisBuilder::Spec`
and we don't support `None` :)